### PR TITLE
Some more cosmetics on the unconference page

### PR DESCRIPTION
**Pull request by bast**

### CODE REVIEW

File: content/events/2021-online-unconference/_index.md [hunk 1]
```diff
@@ -26,30 +26,32 @@ who are or identify as RSEs, but anybody interested in the interface of research
...
 Here are some examples of possible contributions:
 - *Discussion topics*: is software, research software community or research software engineer something close to your heart? Come chatting with other enthusiasts!
-- *Demonstrations:* show us some tools or softwares that you like
+- *Demonstrations:* show us some tools or software that you like
 - *Workshop or [ReproHacks](https://reprohack.github.io/reprohack-hq/)* (only 3 hours in a day)
-- *Talks*: teach us something new 
-- *On going projects*: share with others something unfinished and almost working where you would like to get feedback on
-- *Something else*: something you would like to share with others that does not fit any of the previuous points? We are looking forward to hearing about it!
+- *Talks*: teach us something new
+- *On-going projects*: share with others something unfinished and almost working where you would like to get feedback on
```
Comment (PatricHolmvall) [inline content/events/2021-online-unconference/_index.md:39]:
```suggestion
- *On-going projects*: share with others something unfinished and almost working that you would like to get feedback on
```

File: content/events/2021-online-unconference/_index.md [hunk 2]
```diff
@@ -26,30 +26,32 @@ who are or identify as RSEs, but anybody interested in the interface of research
...
+- *Something else*: something you would like to share with others that does not fit any of the previous points? We are looking forward to hearing about it!
```
Comment (PatricHolmvall) [inline content/events/2021-online-unconference/_index.md:40]:
```suggestion
- *Something else*: is there something you would like to share with others that does not fit any of the previous points? We are looking forward to hearing about it!
```

**Review by PatricHolmvall — APPROVED** [review]
Great changes, feel free to merge after accepting/rejecting suggestions.

**Comment by bast** [comment]
Thanks for the changes!

**Pull request merged**